Cargo.toml

```diff
@@ -237,4 +237,3 @@ polimec-runtime = { path = "runtimes/polimec" }
 asset-hub-polkadot-runtime = { git = 'https://github.com/polkadot-fellows/runtimes', default-features = false, rev = '59172d5fb8feea2646490618bead3b2b4fb06220' }
 polkadot-runtime-constants = { git = 'https://github.com/polkadot-fellows/runtimes', default-features = false, rev = '59172d5fb8feea2646490618bead3b2b4fb06220' }
 system-parachains-constants = { git = 'https://github.com/polkadot-fellows/runtimes', default-features = false, rev = '59172d5fb8feea2646490618bead3b2b4fb06220' }
-polkadot-runtime = { git = 'https://github.com/polkadot-fellows/runtimes', default-features = false, rev = '59172d5fb8feea2646490618bead3b2b4fb06220' }
```

Comment:
Wait, is the dep now removed? Should still keep it for integration tests, right?

Reply:
Thought your comment was to delete this line?

Reply:
I think in the previous version is added a new line after this line. So was a nit to remove the new line. But we should probably still include this line, else integration tests won't compile